src/string_kind.rs

```diff
@@ -26,7 +26,7 @@ impl StringKind {
     Self::new(StringDelimiter::QuoteSingle, false),
   ];
 
-  const fn new(delimiter: StringDelimiter, indented: bool) -> Self {
+  pub(crate) const fn new(delimiter: StringDelimiter, indented: bool) -> Self {
```

Comment:
Can `pub(crate)` be removed now?

Reply:
Yeah. I had this file marked as "viewed" in the pr UI and missed it.